pkg/gitparse/gitparse.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	// defaultDateFormat is the standard date format for git.
-	defaultDateFormat = "Mon Jan 02 15:04:05 2006 -0700"
+	defaultDateFormat = "Mon Jan 2 15:04:05 2006 -0700"
```

Comment:
I believe this is a bug as `git log` appears to use `d` and not `dd` formatting. It's unclear if or how frequently this failure occurred, as the logging in `isDateLine`/`isAuthorDateLine` was level 2.